packages/babel-plugin-syntax-numeric-separator/README.md

```diff
@@ -0,0 +1,35 @@
+# babel-plugin-syntax-numeric-separator
+
+
```

Comment:
To be consistent across package READMEs, could you add there a description:

```md
# babel-plugin-syntax-numeric-separator

> Transforms blablabla

## Installation
[...]
```

We should maybe add this to the spec implementor documentation.